pkg/cmd/admin/policy/modify_scc.go

```diff
@@ -44,8 +44,8 @@ func NewCmdAddSCCToGroup(name, fullName string, f *clientcmd.Factory, out io.Wri
 
 	cmd := &cobra.Command{
 		Use:   name + " SCC GROUP [GROUP ...]",
```

Comment:
undo these changes... need to think about how to phrase this better... the data model really does add users/groups to SCCs